configurator/configurator.go

```diff
@@ -74,6 +66,17 @@ func addConfigFile() {
 }
 
 func setDefaultConf() {
+	var (
+		defaultStartPath              = "."
+		defaultBlackList              = ExtList{"lnk", "ini2", "bin", "tmp"}
+		defaultWhiteListDocs          = ExtList{"doc", "docx", "xls", "xlsx", "ppt", "pptx", "pdf"}
+		defaultWhiteListImgs          = ExtList{"png", "jpg", "jpeg", "raw"}
+		defaultWhiteListExts          = slices.Concat(defaultWhiteListDocs, defaultWhiteListImgs)
+		defaultBlackListFiles         = FileList{"~.ini2"}
+		defaultSizeLimit        int64 = 5_000_000
```

Comment:
Scalar values like numbers, string, runes may be defined with `const`. It perhaps allows compiler to decrease memory usage slightly. In this block `defaultSizeLimit` and `defaultStartPath` may be made constants.

Reply:
could you check it now, pls?

Reply:
It look nice and formatted now! :+1: